samples/benchmarks/coremark/sysbuild.cmake

```diff
@@ -0,0 +1,34 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+if(${images_added})
```

Comment:
Is it actually needed? I think that subimages could actually bring their subimages in generic case (they would not do it for now anyway)

Reply:
Then we have recursive addition of images

Reply:
Recursive from what?

Reply:
There is even a comment :) :
  # same source directory is used for all targets
  # to avoid recursive calls do not add project if remote was already added

We use the same directory for all images so the "remote" image doesn't try to add next images.

Reply:
This should be fixed in sysbuild instead then so it does not recursively source the same file @tejlmand

Reply:
It actually seems like a sysbuild issue (looks like the `sysbuild.cmake` is executed twice with board set to `nrf5340dk_nrf5340_cpuapp`. For now, we could keep it as is but we need to report this and remove workaround as soon as it's fixed.